src/styles/Header.js

```diff
@@ -0,0 +1,16 @@
+import React, {PropTypes} from 'react';
```

Comment:
Yes it's repeated, I'm going to remove it, but the intention is to added in the future pr's